gem/templates/reset_password.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+    <div class="default-container">
+        <div class="title">
+            <h3>{% trans "Reset PIN" %}</h3>
+        </div>
+        <div class="block">
+            <form method="post">
+                {% csrf_token %}
+
+                {% if form.errors %}
+                    <p class="error errorlist">
+                    {% for key, errors in form.errors.iteritems %}
+                        {% for error in errors %}
+                            {% trans error %}<br/>
+                        {% endfor %}
+                    {% endfor %}
+                    </p>
+                {% endif %}
+
+                <p><strong>{% trans "PIN" %}</strong></p>
+                {{ form.password }}
+
+                <p><strong>{% trans "Confirm PIN" %}</strong></p>
+                {{ form.confirm_password }}
+
+                <br>
+                <input type="submit" value='{% trans "RESET PIN" %}' />
```

Comment:
I think the case for "Reset PIN" needs to match `line 7` and any uppercasing should be added in the CSS if needed.